arkenv/src/main/kotlin/com/apurebase/arkenv/ArkenvBuilder.kt

```diff
@@ -9,6 +9,11 @@ import com.apurebase.arkenv.feature.cli.CliFeature
  */
 class ArkenvBuilder(installAdvancedFeatures: Boolean = true) {
 
+    /**
+     * A common prefix that is applied to all argument names when not blank.
+     */
+    var commonPrefix = ""
```

Comment:
Wouldn't it be more self explanatory if this was `var commonPrefix: Int? = null`

Reply:
Was thinking about that. I'll update.